model/configuration.py

```diff
@@ -266,13 +266,14 @@ def with_setting_value(cls, _db, protocol, goal, key, value):
         made unique by a ConfigurationSetting, such as
         ExternalIntegration.URL, rather than by anything in the
         ExternalIntecation itself.
+
         :param protocol: ExternalIntegrations must have this protocol.
         :param goal: ExternalIntegrations must have this goal.
         :param key: Look only at ExternalIntegrations with
             a ConfigurationSetting for this key.
         :param value: Find ExternalIntegrations whose ConfigurationSetting
             has this value.
-        :return: A Query object.
+        :returns: A Query object.
```

Comment:
I think `:return:` is correct -- it's all over this code.

Reply:
Either one works, but I'll change to `:return:` for consistency.